tools/cli/go-whisk-cli/commands/namespace.go

```diff
@@ -119,8 +119,10 @@ var listCmd = &cobra.Command{
 }
 
 func init() {
+    listCmd.Flags().BoolVarP(&flags.namespace.date, "sort", "t", false, wski18n.T("sort by date"))
```

Comment:
Looks like this flag will work for `wsk list`, but not for individual list commands like `wsk action list`.